src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricSerializer.cs

```diff
@@ -25,18 +25,18 @@ namespace Dynatrace.OpenTelemetry.Exporter.Metrics
 {
     public class DynatraceMetricSerializer
     {
-        private readonly string prefix;
-        private readonly IEnumerable<KeyValuePair<string, string>> tags;
+        private readonly string _prefix;
+        private readonly IEnumerable<KeyValuePair<string, string>> _dimensions;
```

Comment:
Should be `defaultDimensions` or the like to distinguish them from the dynamic ones coming from the data points.